src/stashcache.py

```diff
@@ -219,17 +223,16 @@ def fetch_ligo_authz_list_if_needed():
         authfile += f"# {id_to_str[authfile_id]}\n"
         authfile += f"{authfile_id} {paths_acl}\n"
 
+    authfile += end_message
     return authfile
 
 
 def generate_public_cache_authfile(global_data: GlobalData, fqdn=None, legacy=True, suppress_errors=True) -> str:
     """
     Generate the Xrootd authfile needed for public caches.  This contains public data only, no authenticated data.
     """
-    if legacy:
-        authfile = "u * /user/ligo -rl \\\n"
-    else:
-        authfile = "u * \\\n"
+    _ = legacy
```

Comment:
So i guess legacy is just a no-op now

Reply:
In this function, yes.  It's used in generate_cache_authfile() and there's some code that depends on generate_public_cache_authfile() and generate_cache_authfile() having the same signature.

Reply:
ok makes sense!